playbooks/bootstrap_node.yml

```
@@ -0,0 +1,152 @@
+# ==============================================================================
+# [TOOL] DSOM Node Bootstrapper (v1.0)
+#
+# Logic: Dual-Play transition for secure identity bootstrapping across clusters.
+# Play 1: Rootful Bootstrap (Target Initial User -> dsom-admin creation)
+# Play 2: Sovereign Preparation (Target dsom-admin -> Full Configuration)
+# ==============================================================================
+
+# --- [ PLAY 1: IDENTITY BOOTSTRAP (ROOTFUL) ] ---
+- name: "DSOM | Bootstrap Node Identity"
+  hosts: "{{ target_host | default('all') }}"
+  tags: [ bootstrap, setup ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)